apps/Deskreen/install-32

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+mkdir .Deskreen || error 'Failed to make dir ".Deskreen"!'
+cd .Deskreen || error 'Failed to cd into .Deskreen!'
+#Download deskreen-1.0.1-armhf.deb from Discord
+wget https://cdn.discordapp.com/attachments/769219853219004456/803673926554484736/Deskreen-1.0.1-armv7l.AppImage.zip || error 'Failed to download "Deskreen-1.0.1-armv7l.AppImage.zip" from Discord!' 
```

Comment:
Let's upload the appimage to a proper hosting site, rather than to Discord. That way, script-readers can see who "owns" the appimage.

Reply:
Ok. Do you have any recommendations for a host?

Reply:
> Ok. Do you have any recommendations for a host?

Github should be the best choice.

Reply:
Here is a mediafire link, I don't know if it will stay active:

[https://download1081.mediafire.com/6002xvf5l3jg/m2jnyl2mjle7ir5/Deskreen-1.0.1-armv7l.AppImage.zip](https://download1081.mediafire.com/6002xvf5l3jg/m2jnyl2mjle7ir5/Deskreen-1.0.1-armv7l.AppImage.zip)

Reply:
Looks like the MediaFire link still works, here is one from OSDN, but for me it's super slow:

[https://osdn.dl.osdn.net/deskreen-armhf-mirror/76243/Deskreen-1.0.1-armv7l.AppImage.zip](https://osdn.dl.osdn.net/deskreen-armhf-mirror/76243/Deskreen-1.0.1-armv7l.AppImage.zip)